CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+
+- Add Eslint rules to enforce code style and fix Issues.
```

Comment:
I can include "eslint-mdx" but it will make the rules more stricter and will increase the error counts to go above 2000. Both me and kyu decided we can have these rules for now and we can always add more rules later after we have already fixed most of the errors.